R/board_rsconnect.R

```diff
@@ -587,7 +592,18 @@ update_cache <- function(path, key, value) {
 
 rsc_path <- function(board, path) {
   board_path <- httr::parse_url(board$url)$path
-  paste0("/", board_path, "/", path)
+  file.path(board_path, path, fsep = "/")
```

Comment:
This is necessary because sometimes there is a trailing `/` in `board_path` and sometimes not.

Reply:
And what does `file.path()` do?

Reply:
Hmm.  I was wrong.  I thought `file.path()` does the "right" thing, but it doesn't.  Thanks for pointing this out.  Let me ponder on the best way of fixing this.

Reply:
It looks like it will strip a trailing `/`, but only on windows: https://github.com/wch/r-source/blob/1cd91c2156bdcd8d5c3677b6b3aa00b298ee8085/src/main/paste.c#L424-L433. In general, I don't trust `file.path()` because of this sort of platform specific behaviour.